src/openrct2/ride/TrackData.cpp

```diff
@@ -6656,6 +6666,7 @@ static constexpr TrackDefinition TrackDefinitions[] =
     { TRACK_SLOPE_CURVE_LARGE_BANKED,               TrackPitch::Down25, TrackPitch::Down25, TrackRoll::Right,       TrackRoll::Right,    -16 }, // TrackElemType::RightEighthBankToOrthogonalDown25
     { TRACK_DIAG_BRAKES,            TrackPitch::None,           TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0  }, // TrackElemType::DiagBrakes
     { TRACK_DIAG_BLOCK_BRAKES,      TrackPitch::None,           TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0  }, // TrackElemType::DiagBlockBrakes
+    { TRACK_FLAT,                   TrackPitch::Down25,        TrackPitch::Down25,        TrackRoll::None,        TrackRoll::None,        0                  },  // TrackElemType::SlopedBrakes
```

Comment:
some whitespace shifting is needed

Reply:
This was also outdated code. The whitespace in this block is all a bit wonky, but I tried to make mine match the lines above it.